src/main/java/de/tum/in/www1/artemis/web/rest/ModelingExerciseResource.java

```diff
@@ -126,6 +129,8 @@ public ModelingExerciseResource(ModelingExerciseRepository modelingExerciseRepos
         // validates general settings: points, dates
         exerciseService.validateGeneralSettings(modelingExercise);
 
+        modelingExercise.setKnowledge(modelAssessmentKnowledgeService.createNewKnowledge());
```

Comment:
maybe this needs a comment, if I am not aware of the "knowledge concept" I would not know, what is going on here. It's not as self-explanatory as `notifyTutorGroupAboutExerciseCreated`